tsdb/engine/tsm1/tombstone.go

```diff
@@ -17,11 +17,17 @@ const (
 )
 
 type Tombstoner struct {
-	mu sync.Mutex
+	mu sync.RWMutex
 
 	// Path is the location of the file to record tombstone. This should be the
 	// full path to a TSM file.
 	Path string
+
+	// cache of the stats for this tombstone
+	fileStats []FileStat
```

Comment:
I'm not worried about it being modified, it's read-only.  Re-using the slice is a better idea than setting it to nil too.